src/tests/adding.md

```diff
@@ -0,0 +1,273 @@
+# Adding new tests
+
+**In general, we expect every PR that fixes a bug in rustc to come
+accompanied with a regression test of some kind.** This test should
```

Comment:
Super nitpicky, but "accompanied by" is more common. 🤓 
https://ell.stackexchange.com/a/109564